docs/en/observability/whats-new.asciidoc

```diff
@@ -3,21 +3,129 @@
 
 Here are the highlights of what's new and improved in {minor-version}.
 
-coming[7.13.0]
-
-// Other versions:
-// {observability-guide-all}/7.12/whats-new.html[7.12] |
-// {observability-guide-all}/7.11/whats-new.html[7.11] |
-// {observability-guide-all}/7.10/whats-new.html[7.10] |
-// https://www.elastic.co/blog/whats-new-elastic-observability-7-9-0-unified-agent-kpi-overview-dashboard[7.9] |
-// https://www.elastic.co/blog/elastic-observability-7-8-0-released[7.8] |
-// https://www.elastic.co/blog/elastic-observability-7-7-0-released[7.7] |
-// https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
-// https://www.elastic.co/blog/elastic-observability-7-5-0-released[7.5] |
-// https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
+Other versions:
+{observability-guide-all}/7.13/whats-new.html[7.13] |
+{observability-guide-all}/7.12/whats-new.html[7.12] |
+{observability-guide-all}/7.11/whats-new.html[7.11] |
+{observability-guide-all}/7.10/whats-new.html[7.10] |
+https://www.elastic.co/blog/whats-new-elastic-observability-7-9-0-unified-agent-kpi-overview-dashboard[7.9] |
+https://www.elastic.co/blog/elastic-observability-7-8-0-released[7.8] |
+https://www.elastic.co/blog/elastic-observability-7-7-0-released[7.7] |
+https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
+https://www.elastic.co/blog/elastic-observability-7-5-0-released[7.5] |
+https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
+
+[discrete]
+== Give {minor-version} a try
+
+Try {minor-version} now by deploying {es} and {kib} on
+{ess-product}[Elastic Cloud] or
+by https://www.elastic.co/start[downloading them].
 
 // tag::whats-new[]
+[discrete]
+== {agent}, {fleet}, and Integrations now generally available (GA)
+
+To make it easier to integrate your systems and applications with the {stack}, we are excited to announce
+a major improvement in the 7.14 release.
+
+First, we are launching the generally available (GA) release of our {fleet-guide}/elastic-agent-installation-configuration.html[{agent}],
+a single, unified agent for Observability and Security. With fewer things to configure and install,
+a single agent will simplify data onboarding.
+
+Second, we are launching the GA release of {fleet-guide}/fleet-overview.html[{fleet}], which lets you centrally manage an entire fleet
+of {agent}s at scale. It acts as a control plane that gives you a real-time view of the agent status. In addition, it allows
+you to upgrade {agent}s remotely, execute queries on each host, and also contains security threats.
```

Comment:
```suggestion
you to upgrade {agent}s remotely, execute queries on each host, and also contain security threats.
```

OK so this is actually a really funny typo because `and also contain security threats` means something very different from `and also contains security threats` 😂

Reply:
Oops! Great spot. Thanks.